<launch>
    <param name="file_name" type="string" value="/home/mrk/Experiment/carla0.9.11-yuquan-notrees/bag/carla.bag" />
    <!-- <node pkg="read_bag" type="readTF" name="readTF" output="screen" /> -->
    <node pkg="read_bag" type="readPointCloud" name="readPointCloud" output="screen" />

</launch>